### ISSUE TITLE
s3 bucket no longer exists

### ISSUE THREAD

**Issue by iskandari** [adokter/bioRad#629]
Radar csv is no longer accessible via s3, which results in `download_vpfiles()` failing.  Confirmed with aws cli:

```
aws s3 ls s3://lw-enram/ --no-sign-request --region eu-west-1
An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
```

https://github.com/adokter/bioRad/blob/d4935eddaa7cc1c3c50e47278e72967c8bbd980c/R/utils.R#L62-L64

@peterdesmet do you know if this bucket/file was moved to a different address? Should we use odim codes from [this json](https://raw.githubusercontent.com/enram/aloftdata.eu/main/_data/OPERA_RADARS_DB.json) instead?

**Comment by peterdesmet** [comment]
Yes, that bucket has been removed. It was deprecated for a long time, I didn’t realize this function was still using it.

And indeed, the json file you reference is the best alternative. It’s the file we maintain for  https://aloftdata.eu/radars/

**Comment by iskandari** [comment]
resolved in bbde9f8
